accessibility_monitoring_platform/apps/cases/views.py

```diff
@@ -387,6 +391,13 @@ class CaseDetailUpdateView(CaseUpdateView):
     form_class: Type[CaseDetailUpdateForm] = CaseDetailUpdateForm
     template_name: str = "cases/forms/details.html"
 
+    def form_valid(self, form: ModelForm):
+        """Process contents of valid form"""
+        case: Case = self.object
+        if "enforcement_body" in form.changed_data and case.published_report_url:
+            report_data_updated(audit=case.audit)
+        return super().form_valid(form=form)
+
```

Comment:
Nicely done. Can we extend this to `Full URL` and `Organisation name`?

Reply:
Sure. 71dad9734fdfc65b135393e6c178b4d28288e31c